### DISCUSSION TITLE
Unknown option `-format'

### DISCUSSION THREAD

**Discussion by TAO152** [neperfepx/neper#845] [Q&A] [answered]
When I use the following code：
neper -T -n 100
neper -V n100-id1.tess -format inp
result：
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.6.0
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/m/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -V loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] n100-id1.tess -format inp
Info   : ---------------------------------------------------------------
Info   :   - Reading arguments...
Error  : Unknown option `-format'.
Error  : Bad arguments!
Abandoned (core dumped)
May I ask what problem occurred during the time, which is really troubling, looking forward to your answer

**Comment by rquey** [comment, marked as answer]
I think you meant `neper -M n100-id1.tess -format inp`, right? -V does not have inp output.
